web/info/src/components/nav/theme-button.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+  import { Menu, MenuButton } from '@rgossiaux/svelte-headlessui';
+  import { APP_THEMES } from '$lib/constants';
+  import themeWritable from '$stores/theme';
+
+  let theme: number = APP_THEMES.light;
+
+  themeWritable.subscribe((value) => {
+    theme = value;
+  });
```

Comment:
You can use the auto-subscription syntax of `$theme` here instead - https://svelte.dev/tutorial/auto-subscriptions

Reply:
Thank you so much, this is my first time using svelte!